Gemfile

```diff
@@ -23,23 +21,37 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'bootstrap-sass'
+
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
 # Use Unicorn as the app server
-# gem 'unicorn'
+gem 'unicorn'
+gem 'devise'
+gem 'font-awesome-sass'
+gem 'annotate'
+gem "figaro"
```

Comment:
I have never used dotenv, and since I used figaro in the past, this is the first thing that came to my mind